core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic/dot-experiments-configuration-traffic.component.ts

```diff
@@ -43,6 +45,7 @@ export class DotExperimentsConfigurationTrafficComponent {
         trafficProportion: TrafficProportion;
         trafficAllocation: number;
         status: StepStatus;
+        isExperimentADraft: boolean;
```

Comment:
```suggestion
        state: 'DRAFT' || 'XXXXX'
```

Maybe?

Reply:
in this case, only cares if the state is DRAFT since that define a few things in the UI, in this case basically we can only modify an experiment configuratio that is in draft.